src/sagemaker/feature_store/feature_group.py

```diff
@@ -537,6 +538,66 @@ def describe(self, next_token: str = None) -> Dict[str, Any]:
             feature_group_name=self.name, next_token=next_token
         )
 
+    def update(self, feature_additions: Sequence[FeatureDefinition]) -> Dict[str, Any]:
```

Comment:
`feature_definitions` is one field in FeatureGroup class. Do we need to append the new features in this field?

Reply:
Actually after a second thought, I think let's skip this.  Because, that is also true for `update_feature_metadata`. Plus, if we append what's the value? In any way, this feature definition is used in backend for schema validation, in the studio side, customers are not actively using the definitions for any purpose.

Reply:
FeatureDefinition only has FeatureName and FeatureType. UpdateFeatureMetadata should not update this field. We should append the new features in this field. In studio, what if customer wants to know what features this FG has after calling UpdateFeatureGroup?